.azure/pipelines/components-e2e-tests.yml

```diff
@@ -27,6 +27,8 @@ variables:
   value:  AspNetCore
 - name: npm_config_cache
   value: src/submodules/Node-Externals/cache
+- name: PUPPETEER_SKIP_DOWNLOAD
+  value: 1
```

Comment:
Puppeteer is only used by SignalR, so it's fine if its disabled here "forever".

Reply:
The functional tests do.

Reply:
![image](https://github.com/dotnet/aspnetcore/assets/6995051/3f88bd47-70c4-4761-89cd-840198fce80d)

You use Karma, and that uses puppeteer